app/Http/Controllers/Admin/helpdesk/SocialMedia/SocialMediaController.php

```diff
@@ -103,6 +104,7 @@ public function services()
             'twitter',
             'linkedin',
             'bitbucket',
+            'openId_connect',
```

Comment:
```suggestion
            'openid_connect',
```

Would do the same for all other occurrences

Reply:
It's ok, I rename all these occurrences and squash all commits